anaconda_anon_usage/tokens.py

```diff
@@ -13,6 +13,7 @@
 from .utils import _debug, _random_token, _saved_token
 
 Tokens = namedtuple("Tokens", ("version", "client", "session", "environment"))
+CONFIG_DIR = expanduser("~/.conda")
```

Comment:
I'm wondering if we should use something like https://github.com/platformdirs/platformdirs to enable roaming profiles on Windows etc?

Reply:
Yes, maybe create a separate issue to duplicate conda's own conventions for this directory.